src/main/java/control/DetailViewController.java

```diff
@@ -0,0 +1,119 @@
+package control;
+
+import data.ScriptingProject;
+import gui.CameraShotBlock;
+import gui.DetailView;
+import gui.ShotBlock;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+
+/**
+ * Created by Bart.
+ */
+public class DetailViewController {
+
+    private DetailView detailView;
+    private ControllerManager manager;
+    private ScriptingProject project;
+
+    /**
+     * Constructor.
+     * @param manager - the controller manager this controller belongs to
+     */
+    public DetailViewController(ControllerManager manager) {
+        this.detailView = manager.getRootPane().getRootHeaderArea().getDetailView();
+        this.manager = manager;
+        this.project = manager.getScriptingProject();
+        initDescription();
+        initName();
+        initBeginCount();
+        initEndCount();
+    }
+
+    /**
+     * Init the begincount handlers.
+     */
+    private void initBeginCount() {
+        detailView.setBeginCount(0);
+
+        detailView.getBeginCountField().textProperty().addListener((
+                observable, oldValue, newValue) -> {
```

Comment:
The starting `(` is part of `observable, oldValue, newValue)`, this should be together.